src/components/Table/filters.js

```diff
@@ -5,8 +5,8 @@ function CreateDropDownListHelper(options, filterValue, setFilter, id) {
     <select
       value={filterValue}
       onChange={e => {
-        setFilter(e.target.value || '');
         localStorage.setItem(id, e.target.value);
+        setFilter(e.target.value || '');
```

Comment:
For my own learning, is this a best practice of updating `localStorage` first and then state? 🤔

Reply:
I swapped the order because the setFilter function for the checkbox filter reads localStorage and needs the value to be up to date. It's probably better to have it save first in general before things are refreshed and recomputed.

Reply:
This makes total sense to me - I forgot we had included this behaviour in #167!